src/components/Views/SchedulingView.vue

```diff
@@ -43,32 +47,41 @@ const getProjectName = () => {
   return `${targetName}_${formattedDate}`
 }
 
-const sendObservationRequest = async () => {
+const makeCadenceRequest = (cadenceResponse) => {
+  cadencePayload.value = cadenceResponse
+  observationData.value.isCadenceRequest = false
+  sendObservationRequestOrBuildCadencePayload()
```

Comment:
I see what you are doing here - it might be easier to read if you separate out functions for building a request, and functions for sending a request or getting a cadence. Then your entrypoint function builds your request, checks if its a cadence and sends it to get the cadence - then on success you send the response as a request. If its not a cadence it just sends the request. But by separating the functions you don't have to pass the cadence response back through the sendObservationRequestOrBuildCadencePayload function. The name of the function even suggests its trying to do 2 different things that you could move into separate functions.